src/module.js

```diff
@@ -30,7 +34,9 @@ const getFileTuple = filename => readFile(filename, 'utf-8')
  *
  * @return {Promise}
  */
-export const findModules = () => fileMatching('*.{js,json}');
+export const findModules = () => fileMatching('{*,*/*}.{js,json}')
+    .then(without('index.js'))
```

Comment:
Hmmm, althought it works, it is not correct replace for `a → [a] → [a]`. I'm gonna change that!